craft_providers/lxd/lxd_instance.py

```diff
@@ -556,11 +577,31 @@ def _is_running(timeout: float) -> None:
             error=LXDError(brief="Instance failed to restart."),
         )
 
-    def stop(self) -> None:
+    def stop(self, delay_mins: int = 0) -> None:
```

Comment:
Per conversation with @mr-cal:

Default this to `None` and if it's any integer including zero run the new code path